docs/docs/en/guide/resource/configuration.md

```diff
@@ -2,13 +2,23 @@
 
 The Resource Center is usually used for operations such as uploading files, UDF functions, and task group management. You can appoint the local file directory as the upload directory for a single machine (this operation does not need to deploy Hadoop). Or you can also upload to a Hadoop or MinIO cluster, at this time, you need to have Hadoop (2.6+) or MinIO or other related environments.
 
+## Local File Resource Configuration
+For a single machine, you can choose to use local file directory as the upload directory (no need to deploy Hadoop) by making the following configuration.
+
+### Configuring the common.properties
+
+Configure the file in the following paths: `api-server/conf/common.properties` and `worker-server/conf/common.properties`.
+
+- Change 'data.basedir.path' to the local directory path. Please make sure the directory exists and have read write permissions, such as: `data.basedir.path=C:/tmp/dolphinscheduler`.
```

Comment:
> It's recommanded use it to describe `` Change `data.basedir.path` to the local directory path. Please make sure the directory exists and the user who deploy dolphinscheduler have read and write permissions. For example `data.basedir.path=/tmp/dolphinscheduler` ``.

I think we should specify ```the user who deploy dolphinscheduler have read and write permissions```. Otherwise it may cause misunderstand.

Reply:
> > It's recommanded use it to describe `` Change `data.basedir.path` to the local directory path. Please make sure the directory exists and the user who deploy dolphinscheduler have read and write permissions. For example `data.basedir.path=/tmp/dolphinscheduler` ``.
> 
> I think we should specify `the user who deploy dolphinscheduler have read and write permissions`. Otherwise it may cause misunderstanding.

Okay, I will specify it.